lib/server.js

```diff
@@ -39,8 +39,10 @@ const defaultInflector = { singularize, pluralize };
  * @param {Object} options.factories Server factories
  * @param {Object} options.pretender Pretender instance
  */
-export function createServer(options) {
-  return new Server(options);
+export async function createServer(options) {
+  const server = new Server();
+  await server.config(options);
+  return server;
```

Comment:
Taking the options away from the constructor means you can create a server that isnt configured. Looking at the code, the only thing you need awaited if the interceptor start, which is the very last line of config. What about keeping config non async and instead doing 

```
export async function createServer(options) {
  const server = new Server(options);
  await server.start();
  return server;
```
Move the code from config to its own method, start
```
async start() {
     await this.interceptor.start?.();
}
```

Reply:
Great suggestion, thanks.